dojo/models.py

```diff
@@ -353,6 +353,13 @@ class System_Settings(models.Model):
     mail_notifications_to = models.CharField(max_length=200, default="",
                                              blank=True)
 
+    enable_webhooks_notifications = \
+        models.BooleanField(default=False,
+                            verbose_name=_("Enable Webhook notifications"),
+                            blank=False)
+    webhooks_notifications_timeout = models.IntegerField(default=60,
+                                          help_text=_("How many seconds will DefectDojo waits for response from webhook endpoint"))
```

Comment:
Done: https://github.com/DefectDojo/django-DefectDojo/pull/7311/commits/470585200fdf13deca14bf3db4a54906052ada09
I didn't "commit suggestion" because I need to update migration as well.